packages/logger/README.md

```diff
@@ -10,32 +10,36 @@ Licensed under the MIT License.
 All event names are defined in src/logger-event.ts
 
 -   **HealthCheck:** Sent each time check-health-func runs
--   **BatchPoolStats:** Sent by batch job manager. Custom measurements:
+-   **BatchPoolStats:** Sent by Batch job manager. Custom measurements:
     -   runningTasks: number of running tasks
     -   samplingIntervalInSeconds: the time interval to take pool metrics measurement
     -   maxParallelTasks: the pool configured maximum number of parallel tasks to run
--   **ScanRequestReceived:** Sent when requests received by post-scans-func. Custom measurements:
+-   **ScanRequestReceived:** Sent when request received by post-scans-func. Custom measurements:
     -   totalScanRequests
     -   pendingScanRequests
     -   rejectedScanRequests
--   **ScanRequestsAccepted:** Sent by web-worker each time a batch request is processed and split into multiple separate url scan requests. Custom measurements:
+-   **ScanRequestAccepted:** Sent by web-worker each time received request is processed and split into multiple separate scan requests. Custom measurements:
     -   acceptedScanRequests: number of scans that were accepted for the processing
--   **ScanRequestQueued:** Sent when scan requests are queued for processing. Custom measurements:
+-   **ScanRequestQueued:** Sent when scan request is queued for processing by Batch scan task. Custom measurements:
```

Comment:
NIT: Sent when "a" scan request is queued for processing by Batch scan task. Custom measurements: